src/global.js

```diff
@@ -93,6 +93,10 @@ export class GlobalWebMonetizationState extends EventEmitter {
   }
 
   onMonetizationStop() {
+    if (!document.head.querySelector('meta[name="monetization"]')) {
```

Comment:
This should do the trick, but it's indicating we aren't giving enough information in the events. Hrmmm

Reply:
yeah, maybe there's a difference between stopped and paused? although the fact that we can figure it out from context is good. Different sites might want to decide differently

Reply:
Yeah, that finality we were talking about is exactly this case of a tag being removed. The next tag would have a new requestId